src/Surfnet/StepupMiddleware/CommandHandlingBundle/SensitiveData/SensitiveData.php

```diff
@@ -180,12 +178,13 @@ public static function deserialize(array $data)
 
     public function serialize(): array
     {
+        $vettingType = (!is_null($this->vettingType)) ? $this->vettingType->jsonSerialize() : null;
         return array_filter([
             'common_name'              => $this->commonName,
             'email'                    => $this->email,
             'second_factor_type'       => $this->secondFactorType,
             'second_factor_identifier' => $this->secondFactorIdentifier,
-            'document_number'          => $this->documentNumber,
+            'vetting_type' => $vettingType
```

Comment:
Shouldn't `document_number` not be supported, this seems a BC breaking change to me?

Reply:
Document number is implicitly supported in the vetting type. I believe this is tried and tested. But maybe I'm missing a test scenario here.

My tests included: event replaying with old/new style events. And performed the events from the applications.

Reply:
Seems OK then.